client/src/app/zeebe/Deployment.js

```diff
@@ -233,40 +178,9 @@ export default class Deployment extends EventEmitter {
    * @returns {Promise<Array<Endpoint>>}
    */
   getEndpoints() {
-    return this._config.get(CONFIG_KEYS.ENDPOINTS, []);
-  }
-
-  /**
-   * Set endpoint with given ID.
-   *
-   * @param {Endpoint} endpoint
-   *
-   * @returns {Promise<void>}
-   */
-  async setEndpoint(endpoint) {
-    const endpoints = await this.getEndpoints();
-
-    const index = endpoints.findIndex(({ id }) => id === endpoint.id);
-
-    if (index === -1) {
-      endpoints.push(endpoint);
-    } else {
-      endpoints[index] = endpoint;
-    }
-
-    return this.setEndpoints(endpoints);
+    return this._settings.get(SETTINGS_KEY_CONNECTIONS);
```

Comment:
Looking forward we'd like the connection manager to be source of truth. The deployment should not need to care about settings or endpoints (can be implemented later on).